apps/include/system/binary_update.h

```diff
@@ -55,7 +54,7 @@ typedef struct binary_info_list_s binary_info_list_t;
  * @return 0 (OK) On success. -1 (ERROR) is returned on failure.
  * @since TizenRT v3.0
  */
-int load_new_binary(char *binary_name);
+int update_reload_binary(char *binary_name);
```

Comment:
API naming gets me confused a bit because it has two verbs.
How about 'bu' as a prefix which stands for binary update?